src/main/java/com/iridium/iridiumteams/listeners/SpawnerSpawnListener.java

```diff
@@ -28,8 +28,11 @@ public void onCreatureSpawn(SpawnerSpawnEvent event) {
             if (!teamEnhancement.isActive(spawnerEnhancement.type)) return;
             if (data == null) return;
 
+            int spawnCount = spawner.getSpawnCount();
             spawner.setSpawnCount((spawner.getSpawnCount() * data.spawnMultiplier) + data.spawnCount);
+            if(spawner.getMaxSpawnDelay() != data.spawnDelay) { spawner.setMaxSpawnDelay(data.spawnDelay); }
             spawner.update(true);
+            spawner.setSpawnCount(spawnCount);
```

Comment:
Does this work?

might be worth adding a comment explaining this.

And do we need to do a final .update for when its no longer active?